### PR TITLE
log error a bit differently

### PR DESCRIPTION
**Pull request by Qi77Qi**
Upload fails for some reason.

Not sure why, but log in kibana doesn't show the cause

This is what's in kibana currently
```
 "message": "Feb 23 03:07:47 gce-rawls-dev701 rawls-app[5523]: [ERROR] [08:07:47.175] [rawls-submission-monitor-dispatcher-41896] o.b.d.r.j.SubmissionMonitorActor - Failed to upload metadata for 1c678b38-5e2c-42bd-a33c-4129ebbb0196/7d882ef1-e766-49d1-af71-1c4315cc16f7"
```

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check that the **Product Owner** has signed off on any user-facing changes
- [ ] **Submitter**: Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Database checks:
  * If PR includes new or changed db queries, include the explain plans in the description
  * Make sure liquibase is updated if appropriate
  * If doing a migration, take a backup of the
  [dev](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/backups?project=broad-dsde-dev&organizationId=548622027621)
  and
  [alpha](https://console.cloud.google.com/sql/instances/terraform-r4caezzc35c4tb7pgdhwkmme4y/backups?project=broad-dsde-alpha&organizationId=548622027621)
  DBs in Google Cloud Console
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] Tell your tech lead (TL) that the PR exists if they want to look at it
- [ ] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via Slack and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala [hunk 1]
```diff
@@ -281,7 +281,7 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
           workflowId =>
             uploadMetadataToGCS(workflowRec.submissionId.toString, workflowId, petUser).recover{
               case e =>
-                logger.error(s"Failed to upload metadata for ${workflowRec.submissionId}/${workflowId}", e)
+                logger.error(s"Failed to upload metadata for ${workflowRec.submissionId}/${workflowId} due to ${e}")
```
Comment (dvoet) [inline core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala:284]: no, the old way was the right way

Comment (dvoet) [reply core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala:284]: I think the error information you are looking for is immediately following in the logs

Comment (Qi77Qi) [reply core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala:284]: it should. But I don't see the cause anywhere in kibana..So I'm just changing it to log it as plain text.

Comment (dvoet) [reply core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala:284]: your problem is on line 298 below, you are passing in `None` for the execId which triggers a lookup to cromwell based on labels but it all happens before the labels are set. Look at how the original `ShardedHttpExecutionServiceCluster.callLevelMetadata` function is called as opposed to the new one `callLevelMetadataForCostCalculation`. The code looks in the db for the execId first, you need to do that.

**Review by dvoet — CHANGES_REQUESTED** [review]
this should not be merged, see other comments

**Comment by Qi77Qi** [comment]
closing this PR since the error is indeed logged...it's just in the next message and one can use `View surrounding documents ` link in kibana to see surrounding messages

**Pull request closed without merging**